plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -169,7 +169,7 @@ protected function get_order_item_data( $item ) {
 		// Add SKU and PRICE to products.
 		if ( is_callable( array( $item, 'get_product' ) ) ) {
 			$data['sku']   = $item->get_product() ? $item->get_product()->get_sku() : null;
-			$data['price'] = $item->get_quantity() ? $item->get_total() / $item->get_quantity() : 0;
+			$data['price'] = $item->get_quantity() ? (float) number_format( $item->get_total() / $item->get_quantity(), wc_get_price_decimals(), '.', '' ) : 0;
```

Comment:
Please use the `wc_format_decimal()` method here. See: https://github.com/woocommerce/woocommerce/pull/31593/files#diff-89305498bc7a59db846649f3b8e4e45708eba2903d252d5dbac5b2ef5c1dcfe3R165 for an example.

Reply:
thank you, I modified my code.